src/tests/ftest/util/agent_utils.py

```diff
@@ -151,11 +148,21 @@ def run_agent(basepath, server_list, client_list=None):
             expected_data += output
 
             match = re.findall(pattern, output)
-            if len(match) > 0:
+            if match:
+                started_clients.append(client)
                 print("<AGENT> agent started on node {} in {} "
                       "seconds".format(client, time.time() - start_time))
                 break
 
+        return_code = sessions[client].poll()
+        if return_code is not None:
+            print("Uh-oh, in agent startup, the ssh that started the agent "
```

Comment:
Should we maintain the "<AGENT> ..." message format used in the other print statements?

Reply:
Done